assets/css/icons.scss

```diff
@@ -1,23 +1,12 @@
 @font-face {
 	font-family: job-manager;
-	src: url(../font/job-manager.eot?4963673);
+	src: url(../font/job-manager.eot?16367094);
 	src:
-		url(../font/job-manager.eot?4963673#iefix) format("embedded-opentype"),
-		url(../font/job-manager.woff?4963673) format("woff"),
-		url(../font/job-manager.ttf?4963673) format("truetype"),
-		url(../font/job-manager.svg?4963673#job-manager) format("svg");
-	font-weight: normal;
-	font-style: normal;
-}
-
-@font-face {
-	font-family: jm-logo;
-	src: url(../font/jm-logo/jm.eot?ycsbky);
-	src:
-		url(../font/jm-logo/jm.eot?#iefixycsbky) format("embedded-opentype"),
-		url(../font/jm-logo/jm.woff?ycsbky) format("woff"),
-		url(../font/jm-logo/jm.ttf?ycsbky) format("truetype"),
-		url(../font/jm-logo/jm.svg?ycsbky#icomoon) format("svg");
+		url(../font/job-manager.eot?16367094#iefix) format("embedded-opentype"),
+		url(../font/job-manager.woff2?16367094) format('woff2'),
+		url(../font/job-manager.woff?16367094) format("woff"),
+		url(../font/job-manager.ttf?16367094) format("truetype"),
+		url(../font/job-manager.svg?16367094#job-manager) format("svg");
```

Comment:
There is no need for the extra formats nowadays, `woff2` works everywhere:
https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face#browser_compatibility

Reply:
Good catch! Removed in 770ac8f. Thanks!